packages/gestalt/src/TextArea.css

```diff
@@ -10,3 +10,36 @@
 .textArea::placeholder {
   color: var(--g-colorGray200);
 }
+
+.unstyledTextArea, .textAreaSpacer {
+  composes: unstyled from "./FormElement.css";
+  composes: Text fontSize3 from "./Text.css";
+  line-height: 1.5;
+  resize: none;
+}
+
+.unstyledTextArea {
+  composes: absolute left0 top0 right0 bottom0 from "./Layout.css";
+  height: 100%;
+  padding: 6px 8px;
```

Comment:
We're actually in the process of making Spacing tokens, so in the meantime we should probably stick to our "boint" system (4px = 1 boint), like so:

`padding: calc(var(--g-boint) * 4)`

Also, 6px will not be part of the new system, can this go up to 8 or down to 4?

Reply:
The 6px are there to align the text with the tags. I can't really do any centering hackery because the `<textarea>` needs to have a 100% height. This is what it would look like with 4px:
![Screen Shot 2020-12-18 at 10 17 13 AM](https://user-images.githubusercontent.com/32990998/102630310-62f34f80-411a-11eb-8773-b1a0effdf1f4.png)
![Screen Shot 2020-12-18 at 10 17 22 AM](https://user-images.githubusercontent.com/32990998/102630312-638be600-411a-11eb-8c6c-a7cbc6d933c1.png)
I can change it if @ashleyseto is ok with it

Reply:
I think the issue right now is the text in the input is larger than the text in the tags. Is that correct? I'm not really sure how to resolve this right now. If we want to keep to 4px for now to not add odd spaces, we can. I think most of my spacing comments were around the internal part of the tag so that doesn't look weird.